### ISSUE TITLE
Recaptcha unmount clean-up really does not remove the recaptcha

### ISSUE THREAD

**Issue by pauls-akmentins** [t49tran/react-google-recaptcha-v3#195]
By looking at the cleanup function, I see that the recaptcha badge is removed, `window` object is set to `undefined` and script tags are being removed, but the script it self is still loaded on the site.

Isn't this a legal problem? The script is **not unloaded**, it still continues to collect the data without any disclaimer and via Google Recaptcha docs if we proceed by removing the badge, there somewhere should be some disclaimer, that there is reCaptcha working on the background.

To my understanding, the script tag can be removed, but without reloading the site, it will continue to work in the background and thats not really a proper cleanup.

Has anyone thought about this?

**Comment by wojtekmaj** [comment]
You are right. There's no way to actually "unload" the script. The removal of the script tag that this package does should be considered... Decorative.
